### PR TITLE
Release v1.17.0

### PR DESCRIPTION
**Pull request by andrewsomething**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,14 +66,14 @@ For example, with `wget`:
 
 ```
 cd ~
-wget https://github.com/digitalocean/doctl/releases/download/v1.16.0/doctl-<version>-linux-amd64.tar.gz
+wget https://github.com/digitalocean/doctl/releases/download/v1.17.0/doctl-<version>-linux-amd64.tar.gz
```
Comment (hilary) [inline README.md]: Gah, missed some when I dealt with `README`. Let's just follow the model I started, i.e.,

```
wget https://github.com/digitalocean/doctl/releases/download/v<version>/doctl-<version>-linux-amd64.tar.gz
```

File: README.md [hunk 2]
```diff
@@ -66,14 +66,14 @@ For example, with `wget`:
...
 ```
 
 Or with `curl`:
 
 ```
 cd ~
-curl -OL https://github.com/digitalocean/doctl/releases/download/v1.16.0/doctl-<version>-linux-amd64.tar.gz
+curl -OL https://github.com/digitalocean/doctl/releases/download/v1.17.0/doctl-<version>-linux-amd64.tar.gz
```
Comment (hilary) [inline README.md]:
```
curl -OL https://github.com/digitalocean/doctl/releases/download/v<version>/doctl-<version>-linux-amd64.tar.gz
```

where `<version>` is the full semantic version, e.g., `1.17.0`. 

Then we can stop updating the README each time.

File: README.md (hunk 1 above)
Comment (andrewsomething) [reply README.md]: :+1:

File: README.md (hunk 2 above)
Comment (andrewsomething) [reply README.md]: :+1:

**Comment by hilary** [comment]
:shipit:

**Pull request merged**